conda_env/utils/uploader.py

```diff
@@ -0,0 +1,117 @@
+import os
+from collections import namedtuple
+from conda_env import exceptions
+try:
+    from binstar_client.utils import get_binstar
+except ImportError:
+    get_binstar = None
+
+from binstar_client import errors
+from binstar_client.commands.login import interactive_login
+
+
+ENVIRONMENT_TYPE = 'env'
+
+# TODO: refactor binstar so that individual arguments are passed in instead of an arg object
+binstar_args = namedtuple('binstar_args', ['site', 'token'])
+
+
+class Uploader(object):
+    """
+    Upload environments to Binstar
+    """
+
+    user = None
+
+    @staticmethod
+    def is_installed():
```

Comment:
Style comment: I would keep this as a function.  `conda_env.utils.binstar.is_installed` seems like a good place.